package.json

```diff
@@ -56,6 +53,7 @@
     "webpack": "^3.3.0"
   },
   "dependencies": {
+    "aws-sdk": "^2.390.0",
     "babel-plugin-transform-builtin-extend": "^1.1.2",
```

Comment:
no need in `babel-*` and `dashdash` deps — will make bundle smaller

Reply:
i was having trouble running the function without `babel-runtime` actually.

Reply:
oh, you are right, works without babel